doc/setup.md

```diff
@@ -108,3 +114,10 @@ brakeman
 ```
 
 or just [visit the project on Gemnasium](https://gemnasium.com/18F/C2).
+
+### Re-indexing search
+
+```bash
+rake environment elasticsearch:ha:import NPROCS=2 CLASS="Proposal"
```

Comment:
As the error message suggests, you need to add `FORCE=y` the first time you create an index.